src/main/java/com/ably/kafka/connect/ChannelSinkTask.java

```diff
@@ -123,9 +124,12 @@ public void put(Collection<SinkRecord> records) {
 
                 channel.publish(message);
             } catch (AblyException e) {
-                // The ably client should attempt retries itself, so if we do have to handle an exception here,
-                // we can assume that it is not retryably.
-                throw new ConnectException("ably client failed to publish", e);
+                if(ably.options.queueMessages){
```

Comment:
I recognise we've not got anything like checkstyle installed in CI for this codebase, however please remain consistent with the code around your addition in terms of formatting.

```suggestion
                if (ably.options.queueMessages) {
```

Reply:
https://github.com/ably/kafka-connect-ably/pull/36/commits/ccc84e7470adb572f33ba286d97912b69e6dfffe